Makefile

```diff
@@ -1,35 +1,54 @@
-GOFLAGS :=
-IMAGE_REPOSITORY_NAME ?= openshift
-
-all build:
-	go build $(GOFLAGS) ./cmd/cluster-kube-controller-manager-operator
-.PHONY: all build
-
-verify-govet:
-	go vet $(GOFLAGS) ./...
-.PHONY: verify-govet
-
-verify: verify-govet
-	hack/verify-gofmt.sh
-	hack/verify-codegen.sh
-	hack/verify-generated-bindata.sh
-.PHONY: verify
-
-test test-unit:
-ifndef JUNITFILE
-	go test $(GOFLAGS) -race ./...
-else
-ifeq (, $(shell which gotest2junit 2>/dev/null))
-$(error gotest2junit not found! Get it by `go get -u github.com/openshift/release/tools/gotest2junit`.)
-endif
-	go test $(GOFLAGS) -race -json ./... | gotest2junit > $(JUNITFILE)
-endif
-.PHONY: test-unit
-
-images:
-	imagebuilder -f Dockerfile -t $(IMAGE_REPOSITORY_NAME)/origin-cluster-kube-controller-manager-operator .
-.PHONY: images
+all: build
+.PHONY: all
+
+# Codegen module needs setting these required variables
+CODEGEN_OUTPUT_PACKAGE :=github.com/openshift/cluster-kube-controller-manager-operator/pkg/generated
+CODEGEN_API_PACKAGE :=github.com/openshift/cluster-kube-controller-manager-operator/pkg/apis
+CODEGEN_GROUPS_VERSION :=kubecontrollermanager:v1alpha1
+
+# Include the library makefile
+include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machinery/make/, \
+	operator.mk \
+)
+
+# This will call a macro called "build-image" which will generate image specific targets based on the parameters:
+# $0 - macro name
+# $1 - target suffix
+# $2 - Dockerfile path
+# $3 - context directory for image build
+# It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
+$(call build-image,origin-$(GO_PACKAGE),./Dockerfile,.)
+
+# This will call a macro called "add-bindata" which will generate bindata specific targets based on the parameters:
+# $0 - macro name
+# $1 - target suffix
+# $2 - input dirs
+# $3 - prefix
+# $4 - pkg
+# $5 - output
+# It will generate targets {update,verify}-bindata-$(1) logically grouping them in unsuffixed versions of these targets
+# and also hooked into {update,verify}-generated for broader integration.
+$(call add-bindata,v3.11.0,./bindata/v3.11.0/...,bindata,v311_00_assets,pkg/operator/v311_00_assets/bindata.go)
+
 
 clean:
 	$(RM) ./cluster-kube-controller-manager-operator
 .PHONY: clean
+
+# TODO: move this to library-go converted to `oc adm release new` which handles the substitution.
```

Comment:
delete origin-release here. We should not cargo cult this. It's kind of wrong. https://github.com/openshift/origin/pull/21375 is the right approach.